keyboards/matrix/pjf/rules.mk

```diff
@@ -0,0 +1,53 @@
+## chip/board settings
+# - the next two should match the directories in
+#   <chibios>/os/hal/ports/$(MCU_FAMILY)/$(MCU_SERIES)
+MCU_FAMILY = STM32
+MCU_SERIES = STM32F1xx
+
+# Linker script to use
+# - it should exist either in <chibios>/os/common/ports/ARMCMx/compilers/GCC/ld/
+#   or <this_dir>/ld/
+MCU_LDSCRIPT = pjf
```

Comment:
Is this 100% required, instead of one of the defaults provided in `lib/chibios/os/common/startup/ARMCMx/compilers/GCC/ld`?

Reply:
yes, a custom bootloader was flashed to the mcu(which emulated a usb stick to ease firmware updating).

Reply:
Bootloaders like this seem like a good idea at first, but turn out awful in practice. (see TADA68)
Does this one have protection against easy bricking?